content/2.0.html.haml

```diff
@@ -90,109 +89,115 @@ title: "Jenkins 2.0"
-
-    %p
-      Learn more about this on our
-      %a{:href => '/solutions/pipeline'}
-        Pipeline solutions page
-      !
-
-  %dt
-    Out of the box: Easy to get it right
-  %dd
-    %p
-      For new users, Jenkins 2.0 starts off with set of recommended plugins to
-      help get you started with the right set of tools to get up and running with
-      Jenkins quickly.
-
-      For the more adventurous users, the Jenkins 2.0 initial setup process
-      also allows you to pick and choose exactly the plugins you want to meet
-      your specific needs. The list of plugins is hand-picked by Jenkins experts
-      from our popular plugins to ensure the best getting-started experience for
-      new users.
-
-    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
-      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
+%h2
+  Pipelines
+
+%h3 Problem
+
+%p As organizations of all types seek to deliver high quality software faster, their use of Jenkins is extending beyond just CI to Continuous Delivery (CD) In order to implement CD, teams  need a flexible way to model and orchestrate their entire delivery pipeline.
+  
+%h3 Solution
+
+%p Jenkins 2.0 now natively supports delivery pipelines as a first-class entity. Pipelines introduces a domain specific language that help Jenkins users to model their software delivery pipelines as code, which can be checked in and version controlled along with the rest of your project's source code.
+
+%img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
+
+%p An expressive domain specific language (configuration as code) makes it easier and faster to create a simple "build, test and deploy" pipeline or build advanced pipelines. The net result is improved productivity for dev and ops teams.
+
+%h4 Key Features and Benefits for Pipelines
+
+%table{:border => 1} 
+  %tr
+    %th Features
+    %th Benefits
+
+  %tr 
+    %td Pipeline as code (DSL)
+    %td
+      %ul
+        %li Easily define simple and complex pipelines through the DSL in a <i> Jenkinsfile</i>.
+        %li Pipeline as code provides a common language to help teams (e.g. Dev and Ops) work together.
+        %li Easily share pipelines between teams by storing common functions in global libraries
+        
+  %tr
+    %td New Pipeline Stage View
+    %td 
+      %ul
+        %li A new interface addition to make it easy to visualize the progress across an entire pipeline.
+        %a{:href => '/images/pipeline-stage-view.png', :target => '_blank'}
+          %img{:src => '/images/pipeline-stage-view.thumb.png', :title => 'A pipeline stage view'}
+ 
+  %tr
+    %td Durable
+    %td
+      %ul
+        %li Pipelines are long lasting and can survive infrastructure outages.
+        
+  %tr      
+    %td Native support for Git, GitHub branches
+    %td 
+      %ul
+        %li Each feature branch can have its own delivery pipeline. 
```

Comment:
Will each feature branch really have their own, unique delivery pipelines?
I can already build every feature branch with a "delivery pipeline" just now; I can also use the multibranch plugin or Job DSL to get a new job for each branch — that's just as "native" as the pipeline multibranch plugins.

It would be nicer to explicitly say that jobs can be automatically created and removed when branches are added to or removed from Git, since that's an actual pain point that users have.